docs/standard/modern-web-apps-azure-architecture/choose-between-traditional-web-and-single-page-apps.md

```diff
@@ -35,7 +35,7 @@ Improvements in user experience made possible by SPA model must be weighed again
 
 ## Razor Components
```

Comment:
Can you update the header to match your changes - just replace Razor Components with Blazor? Thanks!